### PR TITLE
Remove disabled tensorflow mnist softmax system test.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review
@baldersheim , @lesters : FYI

### CODE REVIEW

**Comment by geirst** [comment]
@lesters This test has been disabled since 2022-06-12, and I guess the functionality it tests is no longer relevant? (_"TensorFlow and Onnx model should produce equals results"_).

**Review by geirst — APPROVED** [review]
👍  @lesters please review as well
